src/api/types.ts

```diff
@@ -308,3 +308,20 @@ export interface UserPermissionsResponse {
 export enum Permission {
   EDIT_SETTINGS = 'EDIT_SETTINGS',
 }
+
+export interface JobsResponse {
+  jobs: Job[];
+}
+
+export interface Job {
+  status: JobStatus;
+  dateCreated: string;
+}
+
+export enum JobStatus {
+  InProgress = 'IN_PROGRESS',
+  Done = 'DONE',
+  Cancelled = 'CANCELLED',
```

Comment:
This is AMERICA 🇺🇸 . Keep your British spelling away from here 😤 
```suggestion
  Canceled = 'CANCELED',
```

Reply:
the colonization has spread far and wide in our codebase 😓, returning from the backend like this right now -- but will update the display text here!

Reply:
Sad 😿  but alright